app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuEnhancedTrackingProtectionRobot.kt

```diff
@@ -143,15 +143,15 @@ private fun assertEnhancedTrackingProtectionOptions() {
     onView(withText("Standard (default)"))
         .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
 
-    val stdText = "Blocks fewer trackers. Pages will load normally."
+    val stdText = "Balanced for privacy and performance. Pages load normally."
```

Comment:
Using `getString` will be ideal but it will require a much bigger refactor as we don't have a `context` reference, for now it could be improve a bit by reusing the `string`.

Reply:
[withText](https://developer.android.com/reference/androidx/test/espresso/matcher/ViewMatchers#withText(int))  accepts a string id 🙌🏽